models/comments.model.js

```diff
@@ -30,3 +30,15 @@ exports.insertComment = (article_id, username, body) => {
       return rows[0]
     })
 }
+
+exports.checkCommentToDelete = (comment_id) => {
```

Comment:
Is this function checking the comment to be deleted or deleting the comment? You could consider names like:
- removeComment
- removeCommentById
- You could use `deleteCommentById`, you have used `deleteComment` in your controller, this may come down to personal preference whether that's good or bad